mr/src/main/java/org/apache/iceberg/mr/InputFormatConfig.java

```diff
@@ -47,6 +47,9 @@ private InputFormatConfig() {
   public static final String CATALOG = "iceberg.mr.catalog";
   public static final String HADOOP_CATALOG_WAREHOUSE_LOCATION = "iceberg.mr.catalog.hadoop.warehouse.location";
   public static final String CATALOG_LOADER_CLASS = "iceberg.mr.catalog.loader.class";
+  public static final String WRITE_FILE_FORMAT = "iceberg.mr.write.file.format";
```

Comment:
Should we add a TODO here to remind us to rename this class (in a follow-on PR I guess?) or move these constants around a bit later?

Reply:
Added the TODO message.
Thanks for reminding me!